### PR TITLE
Fix some issues with the Gliese 752 system

### PR DESCRIPTION
**Pull request by SevenSpheres**

### CODE REVIEW

**Comment by SevenSpheres** [comment]
I can't merge without approval, so someone needs to approve this - this is just fixing an issue from #61

**Review by 375gnu — APPROVED** [review]

**Comment by SevenSpheres** [comment]
Whenever one problem is fixed, a new one appears... I *think* everything's good now.

**Pull request merged**
